src/mudlet.pro

```diff
@@ -461,6 +468,36 @@ macx: {
 
     # Set the .app's icns file
     ICON = icons/osx.icns
+
+    # possibly unnecessary
+#    QMAKE_LFLAGS += -Wl,-rpath,@loader_path/../Frameworks
```

Comment:
If it showed that this was unnecessary indeed, can you remove it again?

Reply:
Removed